src/common/resources/tags.js

```diff
@@ -0,0 +1,12 @@
+import {
+  TAGS_API,
+} from '../constants';
+import { BASE_RESOURCE } from './base';
+
+// eslint-disable-next-line import/prefer-default-export
+export const tagsResource = {
+  ...BASE_RESOURCE,
+  path: TAGS_API,
+  records: 'tags',
+  clear: false,
```

Comment:
this is something new, could you please specify why it's here?

Reply:
no needed